index.js

```diff
@@ -0,0 +1,58 @@
+import Book from './modules/bookClass.js';
+import LocalStore from './modules/localStore.js';
+import FullBook from './modules/fullBook.js';
+import { DateTime } from './modules/luxon.js';
+
+document.addEventListener('DOMContentLoaded', FullBook.displayBooks);
+
+document.querySelector('#booksForm').addEventListener('submit', (e) => {
+  e.preventDefault();
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+  const book = new Book(title, author);
+  FullBook.addToBooks(book);
+  LocalStore.addBook(book);
+  FullBook.clearField();
+});
+
+document.querySelector('#ourBooksListId').addEventListener('click', (e) => {
+  FullBook.removalBook(e.target);
+  LocalStore.removeBook(e.target.previousElementSibling.textContent);
+});
+
+const dateVisit = document.querySelector('.userDate');
+const setExactTime = () => {
+  const now = DateTime.local();
+  const showDate = now.toFormat('dd LLL yyyy');
+  dateVisit.innerHTML = showDate;
+};
+setExactTime();
```

Comment:
- [ ] [REQUIRED] Good job so far! The current date and `Time` are displayed below the navigation bar, for this requirement, you should use the [Luxon](https://github.com/moment/luxon/blob/master/docs/install.md) library. See [Get the current date and time](https://github.com/moment/luxon/blob/master/docs/tour.md#get-the-current-date-and-time).  You have used Luxon well but you have not implemented the function of displaying Current time, please do it.

![image](https://user-images.githubusercontent.com/63915024/183522659-cf80b223-96fd-437d-be28-3202691bc47f.png)